what-are-bots.md

```diff
@@ -0,0 +1,20 @@
+### WHAT ARE BOTS?
```

Comment:
Since this is the heading of the article. One `#` would be more appropriate. i.e.

# WHAT ARE BOTS?

And for subsequent headings, you can use two `#`. ie.
## A subheading